test/basic/checking-fail-bpl.c

```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
```

Comment:
True; perhaps `fail` was the wrong word to use, because these regressions do not “fail” in the sense that the others labeled by `_fail` do — i.e., by failing assertions.

I have changed the names of these regressions to avoid confusion.